stdlib/public/Atomics/AtomicPointers.swift.gyb

```diff
@@ -0,0 +1,514 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+%{
+  def atomicTypes():
+    return [
+      # Swift                     AtomicStorage              Builtin
+      ("UnsafeMutableRawPointer", "UnsafeMutableRawPointer", "RawPointer"),
+      ("UnsafeMutablePointer",    "UnsafeMutablePointer",    "RawPointer"),
```

Comment:
Why only the mutable variants?

Reply:
What's the point of an immutable atomic value?

Reply:
These are the pointee types, not the pointer types, right? You could certainly have an `UnsafeAtomicPointer<UnsafePointer<ConfigurationCStruct>>`, and swap out configurations that way.

Reply:
`UnsafeRawPointer` and `UnsafePointer` both conform to `NullableAtomic` now.